### ISSUE TITLE
First questline eating all the stuff

### ISSUE THREAD

**Issue by Gemberkoekje** [BevoTechPack/BevoTechPack#890]
Hi there,

Firstly, thanks for all your hard work.

Now, this is more of a personal preference than an issue per say, but I wanted to see other people's opinions.

The first questline is 'You need stuff to start out' - and I totally agree it's useful to have a stack of some stuff to start out. The issue I have is with the fact that you hand it over to the questbook - and all you get back is a lousy T-shirt, and a goodiebag of random items.

I think these kinds of quests are more suited to implicit building quests - e.g. give me 10 wood and I'll give you 20 scaffolding, or whatever.

In short, please consider making these quests "You need to have them in your inventory" quests, instead of "You need to give the items to the questbook" quests.

**Comment by vegeta897** [comment]
I agree in that I found this rather odd and slightly frustrating. I just don't see the logic of it.

**Comment by b0bst3r** [comment]
Fixed in BetaG
